flipperdevices/flipperzero-firmware
# Prevent hacking related backgrounds from being displayed in dummy mode.

# What's new

- I added a dummy_mode flag to the AnimationManager struct.
- The animation manager will only display the hardcoded animation in dummy mode. (Currently flipper in front of TV)

# Verification 

- Activate Dummy Mode.
- Confirm background animation.
- Deactivate Dummy Mode.
- Confirm background animation reversion.

# Checklist (For Reviewer)
